### ISSUE TITLE
Team member advertisement 

### ISSUE THREAD

**Issue by yangsong8** [expertiza/expertiza#421]
This bug is spotted in production branch. If a topic has 2 available slots and taken by Team A and Team B, then Team B posts a advertisement, the advertisement does not show  in sign_up_sheet/signup_topics list.

However, I do not see this bug in the scenario that the number of available slot is 1.

**Comment by yangsong8** [comment]
I checked this again and this bug is still there. rrmercer's fix does not work..

**Comment by yangsong8** [comment]
This issue was part of E1652. However, this issue seems to be fix before hand.
I logged in as a user w/o topic. There is one topic with 2 slots and both are taken. 
![image](https://cloud.githubusercontent.com/assets/4420550/20126389/bdbd9c84-a602-11e6-9fb0-d5f00bcfbc66.png)
Clicking the avertisement icon will bring me to a new page with all the ads.
![image](https://cloud.githubusercontent.com/assets/4420550/20126406/d7b225a6-a602-11e6-9b35-8b2eb3ef72d0.png)
I have tested and this user can respond to both ads.
